CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to USWDS web components
+
+Thank you for your interest in contributing to USWDS web components.
+
+> [!Important]
+>
+> **This repo is currently in beta and is not yet ready for community code contributions.**
+>
+> While the USWDS core team works through big decisions about architecture and style, we ask that you do not submit pull requests. Instead, we encourage you to contribute by your sharing your ideas, concerns, questions, and experiences with the USWDS core team and community.
+>
+> Learn more about how you can provide feedback in the [submitting an issue](#submitting-an-issue) and [starting a discussion](#starting-a-discussion) sections below.
+
+## How can I contribute?
+
+### Submitting an issue
+
+If you have discovered a bug, let us know by [submitting an issue on GitHub](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue). We’ll address it as soon as we are able (but because we are in beta, please be aware that it might be awhile).
```

Comment:
**note:** The comment in parentheses is  intended to set expectations about possible response time. There is probably opportunity to improve the language here.

Reply:
I don't think you need parentheses. What if it's just arranged differently, like:
"If you have discovered a bug, let us know by [submitting an issue on GitHub](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue). We are in beta and may not be able to respond immediately, but we will address as soon as as we are able."

Reply:
Thanks! Updated to match your suggestion in [03a1c5e](https://github.com/uswds/web-components/pull/45/commits/03a1c5eb80846105a69f858c122c723d1212c67b)